### PR TITLE
Add v0.22.2 to releases.md

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

**Review by khrm — APPROVED** [review]
/lgtm

**Review by khrm — COMMENTED** [review]
/hold

File: releases.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ Further documentation available:
 
 ### v0.22
 
-- **Latest Release**: [v0.22.1][v0-22-1] (2023-01-13) ([docs][v0-22-1-docs], [examples][v0-22-1-examples])
-- **Initial Release**: [v0.22.0][v0-22-0] (2022-11-16)
+- **Latest Release**: [v0.22.2][v0-22-2] (2023-02-21) ([docs][v0-22-2-docs], [examples][v0-22-2-examples])
+- **Initial Release**: [v0.22.1][v0-22-1] (2023-01-13)
```
Comment (khrm) [inline releases.md:46]: The initial release is v0.22.0

Comment (savitaashture) [reply releases.md:46]: ah okay if we take that way yes its 22.0

Comment (savitaashture) [reply releases.md]: updated

File: releases.md [hunk 2]
```diff
@@ -42,10 +42,10 @@ Further documentation available:
 
 ### v0.22
 
-- **Latest Release**: [v0.22.1][v0-22-1] (2023-01-13) ([docs][v0-22-1-docs], [examples][v0-22-1-examples])
+- **Latest Release**: [v0.22.2][v0-22-2] (2023-02-21) ([docs][v0-22-2-docs], [examples][v0-22-2-examples])
 - **Initial Release**: [v0.22.0][v0-22-0] (2022-11-16)
 - **End of Life**: 2023-03-15
-- **Patch Releases**: [v0.22.1][v0-22-1] [v0.21.0][v0-22-0]
+- **Patch Releases**: [v0.22.2][v0-22-2] [v0.22.1][v0-22-1] [v0.21.0][v0-22-0]
```
Comment (khrm) [inline releases.md:48]: v0.21.0 should be v0.22.0

Comment (savitaashture) [reply releases.md]: ya
updated

**Review by khrm — COMMENTED** [review]
/lgtm

**Review by khrm — COMMENTED** [review]
/kind misc

**Pull request merged**